src/Portalum.Zvt/ZvtClient.cs

```diff
@@ -53,6 +55,20 @@ public class ZvtClient : IDisposable
         /// </summary>
         public event Action<ReceiptInfo> ReceiptReceived;
 
+        /// <summary>
+        /// When the event is registered it is queries periodically to check if the issue of goods is finished.
+        /// Upon successful completion the payment is stored, otherwise an auto reversal is triggered.
+        /// For possible return values see CompletionInfoStatus.
+        /// </summary>
+        public event Func<CompletionInfo> GetAsyncCompletionInfo;
+
+        /// <summary>
+        /// Raised when the payment was successful, but before it is stored in the PT. After this event the GetAsyncCompletionInfo
```

Comment:
the naming and interface of the async completion is debatable, also as it stands async completion is used when `GetAsyncCompletionInfo` is registered which might be less transparent than a separate function / explicit switch.